packages/botonic-core/src/handoff.js

```diff
@@ -49,14 +49,20 @@ export class HandOffBuilder {
     return this
   }
 
+  withShadowing(shadowing = true) {
```

Comment:
please add this to HandOffBuilder in botonic-core/index.d.ts:
`withShadowing(shadowing = true): HandOffBuilder`